see/image_providers/os_glance.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2015-2016 F-Secure
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.  See the License for the specific language governing
+# permissions and limitations under the License.
+"""Glance image provider.
+
+This provider retrieves the requested image from an OpenStack Glance service if
+it doesn't already exist on the configured target path. Images can be requested
+by name or UUID; if name is requested the latest matching image is retrieved.
+
+provider_parameters:
+    target_path (str): Absolute path where to download the image. If target_path
+                       is a directory, the image's UUID will be used as filename.
+    glance_url (str):  The URL of the OpenStack Glance service to query for the
+                       images.
+    os_auth (dict):    A dictionary with OpenStack authentication parameters as
+                       needed by OpenStack's Keystone client.
+
+"""
+
+import os
+
+from datetime import datetime
+from see.interfaces import ImageProvider
+
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
+
+
+class GlanceProvider(ImageProvider):
+
+    def __init__(self, parameters):
+        super(GlanceProvider, self).__init__(parameters)
```

Comment:
why not use the implicit `super()` here ?
`super().__init__(parameters)`
https://www.python.org/dev/peps/pep-3135/